test/code_size/random_printf_wasm2js.json

```diff
@@ -1,6 +1,6 @@
 {
-  "a.html": 17233,
-  "a.html.gz": 7547,
-  "total": 17233,
-  "total_gz": 7547
+  "a.html": 17199,
+  "a.html.gz": 7525,
+  "total": 17199,
+  "total_gz": 7525
```

Comment:
How is this is a codesize win!?

Is it perhaps move the TLS init code into the global scope?  If so that is kind of crazy.

Reply:
Strange it looks like the code size test is still failing. Maybe this isn't actually a code size win. Will re-try again.